koku/sources/api/view.py

```diff
@@ -210,7 +212,12 @@ def stats(self, request, pk=None):
         account_id = get_account_from_header(request)
         schema_name = create_schema_name(account_id)
         source = self.get_object()
-        manager = ProviderManager(source.source_uuid)
-        tenant = Tenant.objects.get(schema_name=schema_name)
-        stats = manager.provider_statistics(tenant)
+        stats = None
+        try:
+            manager = ProviderManager(source.source_uuid)
+        except ProviderManagerError:
+            pass
```

Comment:
what is the response here? Should we error in some way or return an empty payload
```suggestion
            stats = {}
```

Reply:
Since I set `stats = None` above, it just returns an empty payload. I think returning nothing is good enough. Or I could just add the `provider_linked` flag?

Reply:
sometimes empty payload ` ` is not super friendly if json is the expected response.

Reply:
I made `stats = {}` and added the `provider_linked` flag here as well.

Reply:
👍 works for me